### ISSUE TITLE
react-native-blur is not blurring the empty view

### ISSUE THREAD

**Issue by ajaykumar97** [react-native-community/react-native-blur#273]
react-native-blur is not blurring the empty view. It only blurs the view only if it has some content inside it.
Actually I want to blur the background of the side panel such that the content underneath the side panel looks blurred. Something like:
![side](https://user-images.githubusercontent.com/38132295/41017247-bc3fd762-6971-11e8-9f85-1e3b741d7700.jpg)

**Comment by ajaykumar97** [comment]
Yes that's right. It works perfectly fine for iOS. But does not work for android.
Sorry! I forgot to include that issue is coming in Android application.
If anyone get the solution, please help!!!

**Comment by hirejohnsalcedo** [comment]
I think this library fails to support this feature. I've also tried to achieve this look.
